cdap-security/src/main/java/co/cask/cdap/security/store/FileSecureStore.java

```diff
@@ -80,7 +82,14 @@
     // Get the keystore password
     password = cConf.get(Constants.Security.Store.FILE_PASSWORD).toCharArray();
 
-    keyStore = locateKeystore(path, password);
+    try {
+      keyStore = locateKeystore(path, password);
+    } catch (IOException ioe) {
+      // Throw a runtime exception so that it can be thrown by the injector
```

Comment:
It could but now with `return injector.getInstance(FileSecureStore.class) ` we don't need to throw an unchecked exception. Going back to IOException.